addons/mail/static/src/models/channel_member_list_view.js

```diff
@@ -1,65 +1,72 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
-import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ChannelMemberListView',
     identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
+    lifecycleHooks: {
+        _created() {
+            this.channel.fetchChannelMembers();
+        },
+    },
     recordMethods: {
+        /**
+         * Handles click on the "load more members" button.
+         */
+        async onClickLoadMoreMembers() {
+            this.channel.fetchChannelMembers();
+        },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean|FieldCommand}
          */
-        _computeChannel() {
-            if (this.chatWindowOwner) {
-                return replace(this.chatWindowOwner.thread);
-            }
-            if (this.threadViewOwner) {
-                return replace(this.threadViewOwner.thread);
+        _computeAreAllMembersLoaded() {
+            if (!this.channel) {
+                return clear();
             }
-            return clear();
+            return this.channel.memberCount === this.channel.channelMembers.length;
         },
         /**
          * @private
          * @returns {FieldCommand}
          */
-        _computeOfflineCategoryView() {
-            if (this.channel && this.channel.orderedOfflineMembers.length > 0) {
-                return insertAndReplace();
+        _computeChannel() {
+            if (this.chatWindowOwner) {
+                return replace(this.chatWindowOwner.thread.channel);
+            }
+            if (this.threadViewOwner) {
+                return replace(this.threadViewOwner.thread.channel);
             }
             return clear();
         },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean|FieldCommand}
          */
-        _computeOnlineCategoryView() {
-            if (this.channel && this.channel.orderedOnlineMembers.length > 0) {
-                return insertAndReplace();
+        _computeHasMembers() {
+            if (!this.channel) {
+                return clear();
             }
-            return clear();
+            return this.channel.channelMembers.length > 0;
```

Comment:
this should probably be a field on channel as it depends on no other field of the member list view